csharp/BUILD

```diff
@@ -52,9 +51,96 @@ csharp_library(
         "//csharp/Connection:connection",
         "//csharp:typedb_driver_pinvoke",
     ],
-    visibility = ["//visibility:public"],
+    out = "TypeDB.Driver",
     target_frameworks = target_frameworks,
     targeting_packs = targeting_packs,
+    visibility = ["//visibility:public"],
+)
+
+
+swig_native_nuget_pack(
+    name = "driver-csharp-runtime-pack",
+    id = "TypeDB.Driver.Pinvoke",
+    libs = {
+        ":typedb_driver_pinvoke": "TypeDB.Driver.Pinvoke",
+    },
+    native_libs = ["typedb_driver"],
+    files = {
+        "README.md": "README.md",
+    },
+    target_framework = target_framework,
+    nuspec_template = "TypeDB.Driver.Pinvoke.nuspec",
+    visibility= ["//visibility:public"],
+    platforms = {
+        "@vaticle_bazel_distribution//platform:is_mac_arm64": "osx-arm64",
+        "@vaticle_bazel_distribution//platform:is_mac_x86_64": "osx-x64",
+        "@vaticle_bazel_distribution//platform:is_linux_arm64": "linux-arm64",
+        "@vaticle_bazel_distribution//platform:is_linux_x86_64": "linux-x64",
+        "@vaticle_bazel_distribution//platform:is_windows_x86_64": "win-x64",
+    },
+)
+
+
+nuget_push(
+    name = "driver-csharp-runtime-push",
+    src = ":driver-csharp-runtime-pack",
+    snapshot_url = deployment['nuget']['snapshot'],
+    release_url = deployment['nuget']['release'],
+)
+
+
+nuget_pack(
+    name = "driver-csharp-pack",
+    id = "TypeDB.Driver",
+    libs = {
+        ":driver-csharp": "TypeDB.Driver",
+        "//csharp/Api:api": "TypeDB.Driver.Api",
+        "//csharp/Common:common": "TypeDB.Driver.Common",
+        "//csharp/Connection:connection": "TypeDB.Driver.Connection",
+        "//csharp/Concept:concept": "TypeDB.Driver.Concept",
+        "//csharp/Logic:logic": "TypeDB.Driver.Logic",
+        "//csharp/Query:query": "TypeDB.Driver.Query",
+        "//csharp/User:user": "TypeDB.Driver.User",
+    },
+    files = {
+        "README.md": "README.md",
+    },
+    target_framework = target_framework,
+    nuspec_template = "TypeDB.Driver.nuspec",
+    visibility = ["//visibility:public"],
+)
+
+
```

Comment:
You don't need double line breaks here btw, copy the convention from Java or something

Reply:
='(